var/spack/repos/builtin/packages/z3/package.py

```diff
@@ -18,37 +18,28 @@ class Z3(MakefilePackage):
     version('4.8.8', sha256='6962facdcdea287c5eeb1583debe33ee23043144d0e5308344e6a8ee4503bcff')
     version('4.8.7', sha256='8c1c49a1eccf5d8b952dadadba3552b0eac67482b8a29eaad62aa7343a0732c3')
     version('4.5.0', sha256='aeae1d239c5e06ac183be7dd853775b84698db1265cb2258e5918a28372d4a0c')
-    version('4.4.1', sha256='50967cca12c5c6e1612d0ccf8b6ebf5f99840a783d6cf5216336a2b59c37c0ce')
-    version('4.4.0', sha256='65b72f9eb0af50949e504b47080fb3fc95f11c435633041d9a534473f3142cba')
 
-    phases = ['bootstrap', 'build', 'install']
+    phases = ['cmake', 'build', 'install']
```

Comment:
Why do we have phases listed explicitly here? Wouldn't the ones mentioned in the base class do?

Reply:
Just trying to preserve how it was done in the original package. I have removed the line and retested successfully.